peg-solitaire/src/Main.hs

```diff
@@ -56,7 +57,7 @@ main = mainWidgetWithCss ( $(embedFile "static/css/normalize.css")
         rec gs  <- foldDyn move initialState pos
             pos <- mkBoard gs
             elAttr "h3" ("style" =: "text-align: center") $
-                    dynText =<< mapDyn (\g -> "Score: " ++ (show $ score g)) gs
+                    dynText =<< (return . fmap (\g -> "Score: " <> ((T.pack . show) $ score g))) gs
```

Comment:
Why not just `dynText $ fmap (\g -> "Score: " <> (T.pack . show $ score g)) gs`?

Or if you like point free ;) `dynText $ fmap (mappend "Score: " . T.pack . show . score) gs`?

Reply:
Thank you for improving my code. Question: Do I have to take some additional actions before this PR can be merged?